tests/test_logging/basic_scenes_wirte_stuff.py

```diff
@@ -3,11 +3,6 @@
 # This module is used in the CLI tests in tests_CLi.py.
 
 
-class SquareToCircle(Scene):
-    def construct(self):
-        self.play(Transform(Square(), Circle()))
-
-
```

Comment:
@huguesdevimeux could you make sure this change makes sense? I've sort of lost track of tests

Reply:
It makes sense. Renaming the file will provide more clarity.